assets/js/shortcuts.js

```diff
@@ -33,9 +33,12 @@ function scrollHeaders() {
   const headers = Array.from(
     document.querySelectorAll(":is(h1, h2, h3, h4, h5, h6)[id]"),
   );
-  const allShortcuts = Array.from(
-    document.querySelectorAll("#shortcuts > div"),
-  );
+  const shortcuts = document.getElementById("shortcuts");
+
+  // Exit early if shortcuts container doesn't exist
+  if (!shortcuts) return;
```

Comment:
This check is also repeated below; the difference in this context is that it exists purely as a performance optimization - there's no need for `scrollHeaders` to perform scroll calculations if there's no shortcuts container to update.